manifests/config.pp

```diff
@@ -29,17 +29,23 @@
     $_config = $config
   }
 
+  $ensure_unsafe_registration = $unsafe_registration ? {
+    true  => present,
+    false => absent,
+  }
+
+  ini_setting { "${config_file} register-unsafely-without-email true":
+    ensure  => $ensure_unsafe_registration,
+    path    => $config_file,
+    section => '',
+    setting => 'register-unsafely-without-email',
+    value   => true,
+    require => File[$config_dir],
```

Comment:
It feels to me that this is something that ini_setting ought to do, so I checked and it's there:
https://github.com/puppetlabs/puppetlabs-inifile/blob/a0ef2bdc68dcb01713f1f3e50a44824e053927a3/lib/puppet/type/ini_setting.rb#L155-L157

It was added in https://github.com/puppetlabs/puppetlabs-inifile/commit/a2b46d20f298b2f3a861a542c2894031766e2829 which is version 2.5.0 and higher. If you bump that in `metadata.json` you can leave this line out.

On the other hand, you are just moving things around.

Reply:
Thanks